src/main/java/org/testeditor/fixture/host/net/HelperTool.java

```diff
@@ -0,0 +1,42 @@
+package org.testeditor.fixture.host.net;
+
+import java.io.File;
+
+import org.apache.commons.lang3.SystemUtils;
+
+/**
+ * The HelperTool is just a container for convenience methods to proof
+ * prerequisites for different testcases.
+ */
+public class HelperTool {
+
+    private static String pathS3270;
+
+    /**
+     * @return true if the path to s3270 driver is not null and available, false
+     *         otherwise.
+     */
+    public static boolean isS3270DriverPresent(String s3270Path) {
+        if (s3270Path != null && !s3270Path.isEmpty()) {
+            pathS3270 = s3270Path;
```

Comment:
That's a pretty mean side-effect. You ask a question - is this path valid - and then change the internal static state 💣